config/zones.json

```diff
@@ -156,6 +160,8 @@
     "parsers": {
       "consumption": "AX.fetch_consumption",
       "price": "ENTSOE.fetch_price",
+      "consumptionForecast": "ENTSOE.fetch_consumption_forecast",
```

Comment:
AX is actually in the Swedish zone (so we use the ENTSOE price parser), but is a small island.
Therefore, the consumption/generation forecasts will be inaccurate as they will use the SE ENTSOE domain.
I think we should just remove from here.